packages/devextreme/js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -926,14 +945,21 @@ const TreeViewBase = HierarchicalCollectionWidget.inherit({
         return $nodeContainer.not(':empty').length;
     },
 
+    _getItem: function($node) {
```

Comment:
The function return array of items. Why method is named `_getItem`?  in a single plural

Reply:
i'll change implementationit so it returns only single item